### ISSUE TITLE
Weld surface type not welding to smooth surfaces

### ISSUE THREAD

**Issue by Meowkitty-Hates** [Eggs-D-Studios/wos-issues#285]
### Guidelines

- [X] I checked for duplicate bug reports
- [x] I tried to find a way to reproduce the bug

### Version

Main (Production)

### What happened? What did you expect to happen?

The `weld` surface type cannot weld to `smooth` surface types. For example, two `Iron` parts- one with all surface types being `weld`, and the other with all being `smooth`, will not weld together when loaded ingame. Every other surface type combination (e.g. `universal` - `weld`, `inlet` - `studs`, etc.) works perfectly fine, except for this one combination.

This screws with nearly all of the wiring I've done, since I utilize the `smooth` and `weld` surface types in tandem to allow for less messy wiring, and I'm sure it also messes with others, too. This specific surface type combination worked before the update, so it most definitely has something to do with the recent update.

I'm not at all sure if this is intentional or not, but if it is, it's most definitely a better idea to just revert the change, since it doesn't really provide anything other than headache. If anything about this report is unclear or needs further elaboration, you can message me on discord; greeneyes909 is my username! :3

### How can we reproduce this bug?

1. Set one part's surface types to `smooth`, and another's to `weld`.
2. Connect the two parts together with the `smooth` surface touching the `weld` surface.
3. Load the parts ingame.
4. Suffer as you watch them fall apart in front of your very eyes.

### Frequency

Always

**Comment by hail12pink** [comment]
pretty sure this is intended and replicates behavior before roblox started getting rid of surface support...? not really sure the issue here, just use weld surfaces if you want them to weld..?

**Comment by Hexcede** [comment]
Yeah, this isn't a bug. Smooth surfaces aren't supposed to weld to anything. If you want to weld parts anyways, you need to create a manual `Weld` instance in Studio, e.g. using a plugin or snippet of code. MBTools will serialize this weld data and if the part could be welded (not floating) it will be welded.
